app/scripts/process/release-catalog.sh

```diff
@@ -25,6 +25,10 @@ function release_catalog_download_from_url() {
 	printHeader "Download catalog from ${CATALOG_BASE_URL}/$1.json withCredential=${withCredentials}"
 	response=$(curl -sS ${params} -H "Content-Type: application/json" -v ${CATALOG_BASE_URL}/$1.json 2>/dev/null)
 
+	if [ "$1" = "continuous-release-template" ]; then
+		response=sed "s|\${release-version}|$(date '+%d%m%Y')|g" <<< $response 
```

Comment:
This change will cause bash interpretation error since the value isn't surronded with doublequotes. Here is two suggestion to fix that.
```suggestion
		response=`sed "s|\${release-version}|$(date '+%d%m%Y')|g" <<< $response`
```
or 
```suggestion
		response=$(sed "s|\${release-version}|$(date '+%d%m%Y')|g" <<< $response)
```

Reply:
Good catch. I made changes but did not apply them, Thanks Houssem (y)